src/Tests/SettingsTest.php

```diff
@@ -143,5 +143,10 @@ public function testJavaScriptAndDrupalSettings() {
     $this->assertEqual('account_name_1', $drupalSettings['acquia_lift']['credential']['account_name'], '[testJavaScriptAndDrupalSettings]: JavaScript settings contain credential.account_name.');
     $this->assertEqual(90210, $drupalSettings['acquia_lift']['pageContext']['post_id'], '[testJavaScriptAndDrupalSettings]: JavaScript settings contain pageContext.post_id.');
     $this->assertEqual('an_identity', $drupalSettings['acquia_lift']['identity']['identity'], '[testJavaScriptAndDrupalSettings]: JavaScript settings contain identity.identity.');
+
+    // Assert lift3 settings are inserted on page.
+    $this->assertRaw('acquia_lift:account_id', '[testJavaScriptAndDrupalSettings]: acquia_lift account_id is loaded on the node page.');
+    $this->assertRaw('acquia_lift:site_id', '[testJavaScriptAndDrupalSettings]: acquia_lift site_id is loaded on the node page.');
+
```

Comment:
It looks like we already have the vocabularies, terms, and the config setup. Are we already able to see the field metatags on the page? We should be able to functionally test them here.